app/models/appointment.rb

```diff
@@ -1,4 +1,8 @@
 class Appointment < ApplicationRecord
+  include PgSearch
+
+  pg_search_scope :search, against: [:id, :first_name, :last_name]
```

Comment:
Can you use `%i(...)` instead of the symbol array? It's consistent elsewhere then.